src/jaegertracing/Tracer.h

```diff
@@ -301,6 +303,14 @@ class Tracer : public opentracing::Tracer,
     int _options;
 };
 
+
+// jaegertracing::SelfRef returns a StartSpanOption pointing to the Span which traceID and spanID should become the new Span's IDs
+//
+// See opentracing::SpanReference
+inline opentracing::SpanReference SelfRef(const opentracing::SpanContext* span_context) noexcept {
```

Comment:
considering that there is already a custom type `Reference` (unlike in Go), I think it might be better to go with a static method on that class, `Reference::Self(spanCtx)`

Reply:
opentracing::SpanReference and opentracing::StartSpanOption are declared in opentracing/tracer.h which is not included in jaegertracing/Reference.h
I could do what you requested, but with adding a new - from my perspective unrelated - include. Please let me know if you still want it

Reply:
ok, let's keep it here. OpenTracing includes appear to be poorly thought through, e.g. `SpanReferenceType` is defined in `propagation.h`.